.travis.yml

```diff
@@ -303,6 +303,17 @@ jobs:
       after_success:
         - (cd web-console && travis_retry npm run codecov)  # retry in case of network error
 
+    - name: "Build and test on ARM64 CPU architecture"
+      stage: Tests - phase 2
+      arch: arm64-graviton2
+      dist: focal
+      virt: vm
+      group: edge
+      jdk: openjdk11
+      env:
+        - MAVEN_PROJECTS='core,indexing-hadoop,indexing-service,processing,server,services'
```

Comment:
should probably consider adding `sql` to this set too so we at least have coverage of most core Druid things.

Reply:
OK, I will add it!

Reply:
Done!

Reply:
I've split the ARM64 job into two jobs with two different sets of modules to test.
This is needed because otherwise TravisCI will kill the job if it takes more than 50mins.